doc/changelog.rst

```diff
@@ -19,6 +19,8 @@ PyMongo 4.11 brings a number of changes including:
   until either connection succeeds or a server selection timeout error is raised.
 - Added :func:`repr` support to :class:`pymongo.operations.IndexModel`.
 - Added :func:`repr` support to :class:`pymongo.operations.SearchIndexModel`.
+- Added ``sort`` parameter to
+  :meth:`~pymongo.collection.Collection.update_one` and :meth:`~pymongo.collection.Collection.replace_one`
```

Comment:
Also the bulk UpdateOne/ReplaceOne operations.

Reply:
Done